src/components/details-attribute/details-attribute.tsx

```diff
@@ -39,19 +42,20 @@ interface DetailsAttributeDataProps {
     classLabel?: string;
     value?: string | number;
     classValue?: string;
+    lowercaseValue?: boolean;
```

Comment:
Maybe instead of a boolean we could pass actual textTransform value? For example:
```
textTransform: 'uppercase' | 'lowercase' | ...
```